components/encryption/src/manager/mod.rs

```diff
@@ -287,19 +288,23 @@ impl Dicts {
         Ok(Some(()))
     }
 
-    fn rotate_key(&self, key_id: u64, key: DataKey, master_key: &dyn Backend) -> Result<()> {
+    fn rotate_key(&self, key_id: u64, key: DataKey, master_key: &dyn Backend) -> Result<bool> {
```

Comment:
The returned value doesn't seem to be used.

Reply:
used at L362